packages/babel-plugin-ember-test-metadata/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "babel-plugin-ember-test-metadata",
+  "version": "1.2.3",
+  "description": "A Babel plugin for Ember apps that adds a test's file path to test metadata",
+  "keywords": [],
+  "repository": "https://github.com/babel-plugin-ember-test-metadata/babel-plugin-ember-test-metadata",
+  "license": "MIT",
+  "author": {
+    "name": "Glenn Cueto",
+    "email": "gcueto@linkedin.com"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist/*.js"
+  ],
+  "scripts": {
+    "prepublishOnly": "npm run build",
```

Comment:
are we using `npm` in this project?

Reply:
Nope. Though it doesn't really matter in this case, since it's just running a build. I'll change it to yarn (though it was like this previously).